.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Question
+about: Support request or question relating to Koordinator
+title: "[question]"
+labels: kind/question
+
+---
+
+<!-- Please only use this template for submitting question -->
+
+**What happened:**
```

Comment:
this template looks exactly like `bug-report.md`

Reply:
Uses can not feel free to submit bugs